components/Quiz.tsx

```diff
@@ -0,0 +1,73 @@
+"use client"
+
+import { Button } from "@shadcn/ui/button"
+import { Checkbox } from "@shadcn/ui/checkbox"
+import { Label } from "@shadcn/ui/label"
+import { RadioGroup, RadioGroupItem } from "@shadcn/ui/radio-group"
+
+import { Typography } from "./Typography"
+
+type Answer = {
```

Comment:
**suggestion:** we should be consistent between interface and types. I prefer using types. But lets handle this later. :)

Reply:
Should i just do that now?